requirements.txt

```diff
@@ -1,3 +1,5 @@
 cfenv==0.5.3
 requests==2.25.1
 typing-extensions==4.0.0
+ai-api-client-sdk==2.4.0
+aenum==3.1.12
```

Comment:
Can you please add a comment why we are pinning the version of `aenum`. Nobody will remember later otherwise.